doc/installation_instructions/troubleshooting/offline.rst

```diff
@@ -0,0 +1,56 @@
+Airgapped Systems
+-----------------
+
+SmartSim implictly assumes that dependencies can be retrieved via the Internet.
```

Comment:
does implicitly repeat what assume means?

if not, fix misspelling of implicitly...

```suggestion
SmartSim assumes that dependencies can be retrieved via the Internet.
```